<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWidget</class>
 <widget class="QStackedWidget" name="ProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="upload">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
      <property name="icon">
       <iconset resource="images_rc.qrc">
        <normaloff>:/images/images/upload.png</normaloff>:/images/images/upload.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>Total: %p%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>Current: %p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset resource="images_rc.qrc">
        <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string notr="true">Loading files: %v of %m</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
      <property name="icon">
       <iconset resource="images_rc.qrc">
        <normaloff>:/images/images/upload.png</normaloff>:/images/images/upload.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images_rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>upload</sender>
   <signal>clicked()</signal>
   <receiver>ProgressWidget</receiver>
   <slot>uploadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>ProgressWidget</receiver>
   <slot>cancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelClicked()</slot>
  <slot>uploadClicked()</slot>
 </slots>
</ui>
